config/sync/core.entity_view_display.node.recipe.teaser.yml

```diff
@@ -12,21 +24,63 @@ targetEntityType: node
 bundle: recipe
 mode: teaser
 content:
-  links:
-    weight: 100
+  field_r_cooking_time:
+    type: string
+    weight: 2
+    region: content
+    label: inline
+    settings:
+      link_to_entity: false
+    third_party_settings: {  }
+  field_r_hero_image:
+    type: entity_reference_entity_view
+    weight: 4
+    region: content
+    label: hidden
+    settings:
+      view_mode: recipe_teaser
+      link: false
+    third_party_settings: {  }
+  field_r_highlighted:
+    type: boolean
+    weight: 5
+    region: content
+    label: hidden
+    settings:
+      format: default
+      format_custom_false: ''
+      format_custom_true: ''
+    third_party_settings: {  }
+  field_r_serves:
+    type: number_integer
+    weight: 3
+    region: content
+    label: inline
+    settings:
+      thousand_separator: ''
+      prefix_suffix: true
+    third_party_settings: {  }
+  field_r_summary:
+    type: basic_string
+    weight: 0
+    region: content
+    label: hidden
     settings: {  }
     third_party_settings: {  }
+  field_r_teaser_image:
+    type: entity_reference_entity_view
+    weight: 1
     region: content
+    label: hidden
+    settings:
+      view_mode: recipe_teaser
```

Comment:
You forgot to enable and configure the recipe teaser image style in the media image styles display settings

Reply:
done